subsys/partition_manager/CMakeLists.txt

```
@@ -73,6 +73,10 @@ if (CONFIG_BUILD_WITH_TFM)
   ncs_add_partition_manager_config(pm.yml.tfm)
 endif()
 
+if (CONFIG_PM_RESERVE_HUK_KEY)
```

Could we avoid this kconfig symbol altogether?
```suggestion
if (CONFIG_HUK_LOAD)
```

---

I think that we need this flexibility to flash the key. The CONFIG_HUK_LOAD is enabled in b0 and it will fail booting if it cannot load the HUK. So we need this for applications to be able to flash the HUK. Am I missing something here?

---

I think @oyvindronningstad wants a single flag, which identifies both whether the key must be preserved and loaded.
Why are there two flags, as I assume those must always be in sync, otherwise the system will break, is that correctly understood ?

---

@tejlmand is right, I want a single flag. Can you explain more closely the need for flexibility?

---

@oyvindronningstad  The reason that I created that is that we want later to present an example of how to flash a key and I wanted to have the partition defined in a header rather than hard-code the address of the partition in the application code. Since this will only be used for nRF52840 and many people believe that it makes more sense to remove it, I will remove it.